src/js/api/brand.js

```diff
@@ -491,6 +491,96 @@ ripe.Ripe.prototype.onInitialsP = function(options) {
     });
 };
 
+/**
+ * Returns the result of the server side validation of model configurations
+ * considering a given build's brand, model and version.
+ *
+ * @param {Object} options An object with options, such as:
+ *  - 'brand' - The name of the brand to be considerd when validating
+ *  - 'model' - The name of the model to be considerd when validating
+ *  - 'version' - The target build version to be considerd when validating
+ *  - 'parts' - The parts to be validated
+ *  - 'engraving' - The engraving value to be validated
+ *  - 'size' - The size to be validated
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} Resulting information for the callback execution.
+ */
+ripe.Ripe.prototype.validateModel = function(options, callback) {

```

Comment:
Move this after `getFactory` (same order as ripe-core/src/ripe_core/controllers/api/brand.py)